python/pysmurf/client/command/smurf_atca_monitor.py

```diff
@@ -17,81 +17,125 @@
 
 class SmurfAtcaMonitorMixin(SmurfBase):
 
-    _write_atca_monitor_state = ":Crate:WriteState"
+    _write_atca_monitor_state_reg = ":Crate:WriteState"
 
     def write_atca_monitor_state(self, val, **kwargs):
-
         """
         Writes all current ATCA monitor values to a yml file.
 
         Args
         ----
-        val (str) : The path (including file name) to write the yml file to.
+        val : str
+           The path (including file name) to write the yml file to.
         """
-        self._caput(self.shelf_manager + self._write_atca_monitor_state,
-                    val, **kwargs)
-
-    _board_temp_fpga = 'BoardTemp:FPGA'
+        self._caput(
+            self.shelf_manager + self._write_atca_monitor_state_reg,
+            val, **kwargs)
 
-    def get_board_temp_fpga(self, slot_number=None, atca_epics_root=None, **kwargs):
+    _board_temp_fpga_reg = 'BoardTemp:FPGA'
 
+    def get_board_temp_fpga(
+            self, slot_number=None, atca_epics_root=None, **kwargs):
         """
```

Comment:
should add doc strings here. Also, what is a typical `atca_epics_root`?

Reply:
This function not a part of this PR, but agree it needs a docstring.  Created issue to address https://github.com/slaclab/pysmurf/issues/497.